API/src/main/java/de/brandwatch/minianalytics/queryinput/QueryInputApplication.java

```diff
@@ -0,0 +1,12 @@
+package de.brandwatch.minianalytics.queryinput;
```

Comment:
You probably want to rename the package as well then?

```suggestion
package de.brandwatch.minianalytics.api;
```

Reply:
Yes, done. I was wondering why the mvn install still worked...thats why